todo_app/data/ViewModel.py

```diff
@@ -0,0 +1,39 @@
+from todo_app.data.Item import Item
+
+class ViewModel:
+    def __init__(self, list_items):
+        self._items = list_items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        output_list = []
+
+        for item in self.items:
+            if item.status == "To do":
+                output_list.append(item)
+                
+        return output_list
```

Comment:
Nice! Not a recommendation here, but I always think interesting to see the more concise but arguably less readable list-comprehension approach as a comparison here:
```python
        return [item for item in self.items if item.status == "To do"]
```